modules/oe_theme_helper/src/Plugin/Field/FieldFormatter/AddressInlineFormatter.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_helper\Plugin\Field\FieldFormatter;
+
+use CommerceGuys\Addressing\Locale;
+use Drupal\address\AddressInterface;
+use Drupal\address\Plugin\Field\FieldFormatter\AddressDefaultFormatter;
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Language\LanguageInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+
+/**
+ * Plugin implementation of the 'oe_theme_helper_address_inline' formatter.
+ *
+ * @FieldFormatter(
+ *   id = "oe_theme_helper_address_inline",
+ *   label = @Translation("Inline address"),
+ *   field_types = {
+ *     "address",
+ *   },
+ * )
+ */
+class AddressInlineFormatter extends AddressDefaultFormatter implements ContainerFactoryPluginInterface {
```

Comment:
Since now we don't use anything from the base class `AddressDefaultFormatter` we can also not extend it and extend the default class.